src/Mvc/Mvc.RazorPages/src/PageResult.cs

```diff
@@ -39,7 +39,18 @@ public class PageResult : ActionResult
     /// </summary>
     public int? StatusCode { get; set; }
 
-    /// <inheritdoc />
+    /// <summary>
+    /// Requests the service of
+    /// <see cref="M:Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageResultExecutor.ExecuteAsync(PageContext,PageResult)" />
```

Comment:
Thanks for your PR.
Similar to the other comment that I've left in the other PR, the styling is not consistent with the existing styles.
Specifically, the endings of the comment tags has an extra space whereas the existing comment tags don't.

Reply:
<inheritdoc /> is a comment tag that has an extra space, so I am not sure what you mean.

Reply:
Check out how the `see` tag is used on line 33